recipes/vlc/all/conanfile.py

```diff
@@ -0,0 +1,90 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+import os
+
+
+class VlcConan(ConanFile):
+    name = "vlc"
+    license = "GPL"
+    homepage = "https://www.videolan.org/vlc/"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "VLC is a free and open source cross-platform multimedia " \
+                  "player and framework that plays most multimedia files as " \
+                  "well as DVDs, Audio CDs, VCDs, and various streaming protocols."
+    topics = ("vlc", "multimedia", "sound")
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "pkg_config"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_lua": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_lua": False,
+    }
+
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def system_requirements(self):
+        packages = []
+        if tools.os_info.is_linux and self.settings.os == "Linux":
+            if tools.os_info.with_apt:
+                packages = ["libxcb-composite0-dev", "libxcb-xv0-dev"]
+            else:
+                self.output.warn("Do not know how to install dependencies for {}.".format(tools.os_info.linux_distro))
+        if packages:
+            package_tool = tools.SystemPackageTool(conanfile=self, default_mode="verify")
+            package_tool.install_packages(update=True, packages=packages)
+
+    def requirements(self):
+        if self.options.with_lua:
+            self.output.warn("VLC requires luac which is not included in the CCI lua package")
```

Comment:
```suggestion
            self.output.warn("VLC requires luac which is not included in the CCI lua package")
```

What do you mean by not included? luac is a Lua compiler? Asking because we could update lua package if it's the case. Or is it another project related to lua?

Reply:
https://github.com/lua/luac

It appears it's a separate project, however enabling lua looks for both luac and what is included in the lua package.

Reply:
Interesting I'll take a look.

Reply:
Okay, there is an option for luac in lua conan package, but is disable by default.